src/Config/RectorConfig.php

```diff
@@ -41,7 +42,10 @@ final class RectorConfig extends Container
 
     public static function configure(): RectorConfigBuilder
     {
-        return new RectorConfigBuilder();
+        $rectorConfigBuilder = new RectorConfigBuilder();
+        $rectorConfigBuilder->withSetProviders([new TwigSetProvider()]);
```

Comment:
is this enabled by default?

Reply:
This will be moved to symfony package. But yet, it will be to load the sets with their particular package + version.